cinch/library/jenkins_plugin.py

```diff
@@ -0,0 +1,826 @@
+#!/usr/bin/python
```

Comment:
I know that this file was brought in from another repo, but should we change the shebang to **#!/usr/bin/env python** to match the rest of our code?  Also, would a git submodule be appropriate here?  Just curious.